client/executor/wasmtime/Cargo.toml

```diff
@@ -33,3 +33,6 @@ wasmtime = { version = "0.27.0", default-features = false, features = [
 sc-runtime-test = { version = "2.0.0", path = "../runtime-test" }
 sp-io = { version = "4.0.0-dev", path = "../../../primitives/io" }
 wat = "1.0"
+
+[features]
+jitdump = ["wasmtime/jitdump"]
```

Comment:
Why do we need a feature when a env variable needs to be set to have this enabled?

Reply:
The idea is to be able to activate or deactivate the profiler without having to recompile the whole project.

Reply:
I mean, I don't see any reason for having both. I think that only having the env variable should be enough.

Reply:
If we have only the environment variable it implies that the jitdump feature is always activated when we activate wasmtime, are you ok with that?

Reply:
@bkchr I tend to agree with @librelois, I'm not sure the cost of enabling that feature even without setting the profiler. As it is touching a critical part, I want to make sure we don't include useless cost to it.
Your decision Basti

Reply:
I just checked the code. The feature is normally enabled by default in wasmtime. As we disabled default features, it is currently not enabled. However, I would assume that this feature isn't influencing the runtime as long as the config option isn't set. 

TLDR: I would enable the feature by default in wasmtime and would not forward this to everywhere. 

@pepyakin WDYT?

Reply:
@bkchr agreed!

Reply:
Done